server/app.ts

```diff
@@ -77,10 +82,16 @@ export default function createApp(
   // no authentication
   app.get('/privacy-policy', (req, res) => res.render('pages/privacy-policy/privacy-policy'))
   app.use('/start', setupLegalSenderStartPage())
-  app.use('/link', requestLinkAuthorised())
-  app.use('/link', setUpLink(app, magicLinkService, appInsightsClient))
   app.use('/contact-helpdesk', setupContactHelpdesk(zendeskService))
 
+  if (config.featureFlags.lsjOneTimeCodeAuthEnabled) {
```

Comment:
The feature flag will only enable 1 set of routes, which means this is the only bit of code that needs to know about the feature flag